x-pack/plugins/infra/common/log_analysis/job_parameters.ts

```diff
@@ -6,21 +6,41 @@
  */
 
 import * as rt from 'io-ts';
+import { v5 } from 'uuid';
+import { IdFormat, JobType } from '../http_api/latest';
 
 export const bucketSpan = 900000;
 
 export const categoriesMessageField = 'message';
 
 export const partitionField = 'event.dataset';
 
-export const getJobIdPrefix = (spaceId: string, sourceId: string) =>
-  `kibana-logs-ui-${spaceId}-${sourceId}-`;
+const ID_NAMESPACE = 'f91b78c0-fdd3-425d-a4ba-4c028fe57e0f';
```

Comment:
**nit:** Hardcoded IDs in the codebase are a bit of a code smell in the long run, can we just comment on what it represents and why can we keep it hardcoded? Future ourselves will thank us for this 😄

Reply:
The auto merge went through but to address your concern, I feel the variable name and it's usage is clear enough? 

We need to generate a unique ID for each job so we use `uuid.v5` which accepts a UUID to use as the namespace for the hashing and a name unique in that namespace.